README.md

```diff
@@ -422,6 +422,31 @@ done manually by someone on the Azul team. Ask on Slack to find out more.
 
 [Terra Data Repository]: https://jade.datarepo-dev.broadinstitute.org/
 
+### 3.2.2 Personal account registration
+Personal accounts may need to be registered with SAM to allow access to TDR,
+or for registration as an administrator within the TDR JadeStewards-dev group.
+In order to register your personal email with SAM the following actions need to
+be performed.
+
+1. Ensure you are logged in with the correct Google Account. Run
+
+    ```
+    gcloud auth login
+    ```
+
+    For assistance refer to the Google authorization
+    [documentation](https://cloud.google.com/sdk/docs/authorizing).
+
+2. Register your account with SAM. Run
+
+    ```
+    curl $AZUL_SAM_SERVICE_URL/register/user/v1  -d "" -H 
+    "Authorization: Bearer $(gcloud auth --account $account_id print-access-token)"
+    ```
+
+3. Ask an administrator of the TDR JadeStewards-dev group to authorize your 
```

Comment:
There are three states: absent from the group, present in the group, present and admin of the group